src/types.ts

```diff
@@ -8,7 +8,7 @@ export interface NuxtTestContext {
   rootDir: string
   config: NuxtConfig
   nuxt: {
-    config: NuxtConfig
+    options: NuxtConfig
```

Comment:
Nice find! Maybe we should type it to `NuxtOptions` too?

Reply:
Sure

Reply:
Found, thanks to tests :smile: